lib/customer_repository.rb

```diff
@@ -31,19 +31,23 @@ def find_all_by_attribute(attribute, match)
     customers.select { |customer| customer.send(attribute) == match }
   end
 
-  def find_by_first_name(value)
-    find_by_attribute("first_name", value)
-  end
+  #def find_by_first_name(value)
+    #find_by_attribute("first_name", value)
+  #end
 
-  def find_all_by_first_name(value)
-    find_all_by_attribute("first_name", value)
-  end
+  #def find_all_by_first_name(value)
+    #find_all_by_attribute("first_name", value)
+  #end
 
-  def find_by_last_name(value)
-    find_by_attribute("last_name", value)
-  end
+  #def find_by_last_name(value)
+    #find_by_attribute("last_name", value)
```

Comment:
Incorrect indentation detected (column 4 instead of 2).<br>Missing space after #.